src/main/webapp/script.js

```diff
@@ -61,9 +69,16 @@ function fetchList(queryString) {
       totalCardCount++;
     }); 
     initialDisplay();
-    map.fitBounds(bounds);
+    map.fitBounds(bounds);  
+    removeLoader();
   });
 }
+ 
+function removeLoader() {
+  document.getElementById('loader').style.display = 'none';
+  document.getElementById('loader-container').style.display = 'none';
+  document.getElementById('map').style.opacity = '1';
+}
```

Comment:
nit: I might move the initiateLoader and removeLoader declarations to be one after the other, just for clarity's sake